SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/MinProximityIntersection.cpp

```diff
@@ -77,17 +76,12 @@ bool MinProximityIntersection::testIntersection(Cube& cube1, Cube& cube2)
 
 int MinProximityIntersection::computeIntersection(Cube& cube1, Cube& cube2, OutputVector* contacts)
 {
-    return BaseProximityIntersection::testIntersection(cube1, cube2);
-}
-
-bool MinProximityIntersection::getUseSurfaceNormals(){
-    return useSurfaceNormals.getValue();
+    return BaseProximityIntersection::computeIntersection(cube1, cube2, contacts);
 }
 
-void MinProximityIntersection::draw(const core::visual::VisualParams* vparams)
+bool MinProximityIntersection::getUseSurfaceNormals() const
```

Comment:
I find the name of the method strange. Maybe:
isUseSurfaceNormals  or getUseSurfaceNormalsValue()

Reply:
GitHub formatting is not very clear, but I did not introduced `MinProximityIntersection::getUseSurfaceNormals()` myself. It was already in the code. And I agree with you @epernod